pom.xml

```diff
@@ -0,0 +1,270 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <parent>
+        <groupId>org.wso2</groupId>
+        <artifactId>wso2</artifactId>
+        <version>1.2</version>
+    </parent>
+
+    <groupId>org.wso2.carbon.identity.uaepass.outbound</groupId>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>iidentity-outbound-auth-uaepass</artifactId>
+    <version>1.3.20</version>
```

Comment:
WHy does the versioning starts from 1.3.20 ?

Reply:
fixed with : [37e04d3a42](https://github.com/wso2-extensions/identity-outbound-auth-uaepass/pull/1/commits/37e04d3a420d6eeaea5b7d43686e3ab40ac29180)

Reply:
Changed as 1.0.0